controllers/bentodeployment_controller.go

```diff
@@ -2244,6 +2294,41 @@ func (r *BentoDeploymentReconciler) generatePodTemplateSpec(ctx context.Context,
 		podSpec.Containers = append(podSpec.Containers, extraPodSpec.Containers...)
 	}
 
+	if need_monitor_container {
+		monitorExporterImage := "quay.io/bentoml/bentoml-fluentbit:2.0.6"
+
+		var monitorOptEnvs = make([]corev1.EnvVar, 0, len(monitorExporter.Options))
+
+		for k, v := range monitorExporter.Options {
+			monitorOptEnvs = append(monitorOptEnvs, corev1.EnvVar{
+				Name:  "FLUENTBIT_OUTPUT_OPTION_" + strings.ToUpper(k),
+				Value: v,
+			})
+		}
+
+		podSpec.Containers = append(podSpec.Containers, corev1.Container{
+			Name:  "monitor-exporter",
+			Image: monitorExporterImage,
```

Comment:
We should add `resources` for all containers

Reply:
done